README.md

```diff
@@ -3,39 +3,83 @@
 ![Shakespeare](http://www.gomoxie.com/wp-content/uploads/Hamlet.jpg)
 >To be, or not to be...
 
-## Learning Objectives - The student should be able to..
+## Objectives
 
-* Explain that by making parts of their code *conditional*, they can execute different pieces of code based on certain conditions.
-* Explain that an *if* statement will execute a set of statements **only** if that condition is *true*. 
+1. Describe the meaning of *conditional statements*
+1. Use the `if` statement to conditionally execute a set of statements
+1. Use an `else` clause to provide an alternative set of statements
+1. Chain multiple *if-else* statements together
+
+## Conditional statements
+
+You already learned how to use the `Bool` type and various operators to evaluate logical expressions and compare values:
 
 ```swift
 var age = 19
 
+let isOfDrinkingAge = age > 21 // false, because 19 is not greater than 21
+```
+
+As was briefly mentioned in that lesson, those expressions become powerful using *conditional statements* which allow you to write pieces of code which can be executed (or not executed) based on certain conditions. These are called *control flow* constructs, because they allow you to control the execution flow of your program, allowing your program's output to react to conditions, instead of always providing the same output.
+
+## If statement
+
+The `if` statement makes the execution of a set of statements (also called a *block of code*) following it conditional based on a `Bool` value or expression. The given block of code will be executed *only if* the expression evaluates to `true`:
+
+```swift
+let age = 19
+
 if age < 21 {
-    print("You're not allowed to drink!")
+	print("You are not allowed to drink, yet.")
 }
 
-// prints "You're not allowed to drink!"
+// prints "You are not allowed to drink, yet."
 ```
 
-* Explain that an *if* statement can provide an alternative set of statements, known as an *else clause*, for situations when the *if* condition is *false*.
+You can type this block of code in a new Playground file and experiment with giving `age` different values. You will notice that the `print` statement will no longer be executed once you increase the value of `age` to 21 or higher. 
+
```

Comment:
Having them open this in a Playground file is great. I want them interacting with the content like this. I've seen it a lot with the students here, that if they get stuck in reading a lot and not writing code.. they reach a point where when they are finally writing code.. that they don't know where to begin.